README.md

```diff
@@ -175,30 +175,47 @@ expect(1..10).to cover(3)
 ### Collection membership
 
 ```ruby
-expect(actual).to include(expected)
+# exact order, entire range
```

Comment:
I think "range" is potentially confusing since ruby has a range class and this isn't referring to that.  How about using "collection" instead of "range" here and below?